### DISCUSSION TITLE
Location for Bootmgr.exe to Place In

### DISCUSSION THREAD

**Discussion by Lesita** [ipxe/ipxe#1273] [Q&A]
Using "wimboot" there's an error kept producing on every boot attempt, "FATAL: no bootmgr.exe". The so-called "documentation" for both iPXE and "wimboot" don't tell even a bit of the location it needs to be placed in.

Please provide guidance on this situation. I have extracted the binary from the bootmgr file from ISO image. Now I'm getting close, just need to place the file properly.

**Comment by NiKiZe** [comment]
If you use the search function you should find several similar questions.

Normally bootmgr variants is included in boot.wim, which wimboot extracts, if not you need to initrd the file from the .iso that you probably took the boot.wim from.

As long as you United the file properly, there is no "correct location" this is based on the assumption that you are using iPXE to boot wimboot.
